src/util/argstr.rs

```diff
@@ -27,58 +28,47 @@ impl<'a> ArgStr<'a> {
     }
 
     pub(crate) fn contains_byte(&self, byte: u8) -> bool {
-        assert!(byte.is_ascii());
+        debug_assert!(byte.is_ascii());
 
         self.0.contains(&byte)
     }
 
     pub(crate) fn contains_char(&self, ch: char) -> bool {
-        let mut bytes = [0; 4];
-        let bytes = ch.encode_utf8(&mut bytes).as_bytes();
-        for i in 0..self.0.len().saturating_sub(bytes.len() - 1) {
-            if self.0[i..].starts_with(bytes) {
-                return true;
-            }
-        }
-        false
+        self.to_string_lossy().contains(|x| x == ch)
```

Comment:
I am not sure if this function change is correct.

Reply:
The old one compares the `ch` across the char boundray, which is useless(Currently the inner bytes of OsString can only be either `WTF-8` or `UTF-8`). But I think `to_string_lossy` should be improved here since it's O(n).

Reply:
We use `os_str_bytes` lib created by @dylni. Maybe he wants to take a look at this?

Reply:
Thanks for the ping. The updated code is less performant. It validates the encoding, allocates a new string, and searches in the new string instead of searching in the bytes already stored by `ArgStr`. If `slice` had a `contains_subslice` method, this would be much simpler.